### ISSUE TITLE
Citations don't work in tech document abstracts

### ISSUE THREAD

**Issue by jcarpi** [nasa/nasa-latex-docs#22]
Citations in document abstracts are not displayed correctly. See the sample code below, and the attached generated PDF file.
Note that the citation in the abstract prints to **[nasa-latex-docs]**, as opposed to **[1]**. 

```LaTeX
% Define Class: %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[template=tech-memo]{nasa-latex-docs} 

% Preamble Section: %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


% Add bibliogrpahy file
\addbibresource{test-document.bib}

\docAbstract[
    \cite{nasa-latex-docs}
]

% Main Document Content: %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{document}

\input{tex/sample_content.tex}

\printbibliography

\end{document}
% End of Document %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
```

[test-document.pdf](https://github.com/nasa/nasa-latex-docs/files/8234331/test-document.pdf)

**Comment by Aerovet** [comment]
@jcarpi What version of tex did you build the report with?

**Comment by jcarpi** [comment]
@bawood2 `TeX Version : TeX 3.141592653 (TeX Live 2021)` on MacOS. Is that helpful? Seems like an odd version number...

**Comment by Aerovet** [comment]
Yeah, that helps a lot actually.  Ran into the same problem.  This issue was introduced in the Oct26 timeframe in another project I'm working in.  Cross-checking them, it really doesn't make much sense on why citation in docAbstract doesn't display properly.  I think I've narrowed it down to the version of TeX Live being used.  Just ran a test, and this works fine with TeX Live 2020:


TeX Version : TeX 3.14159265 (TeX Live 2020)

[NASA_Latex_Document.pdf](https://github.com/nasa/nasa-latex-docs/files/8235740/NASA_Latex_Document.pdf)

**Comment by Aerovet** [comment]
@jcarpi Also, the version number looked familiar.  Its the digits of pi, for each version of TeX Live, it looks like append the next digit.

**Comment by jorgepz** [comment]
Hello. I am trying this locally on Version 3.141592653-2.6-1.40.25 (TeX Live 2023/Arch Linux)  and I am getting the non-numbered format in the entire document. However, I guess it might be helpful to set a CI workflow so that you can see the output for a given set of architectures and/or versions in the cloud before doing any fix/change in the master.

In https://github.com/nasa/nasa-latex-docs/pull/25 I am humbly proposing setting the CI (this might correspond in a different issue) and a test document such that before doing any merge the PDF can be visualized as in the artifact on the bottom [here](https://github.com/jorgepz/nasa-latex-docs/actions/runs/6242902008). In this case the output is the one mentioned by @jcarpi .
